views/repo_structure.erb

```diff
@@ -11,16 +11,101 @@
           the most recently updated version of its data files.
         </p>
         <p>
-          But if you’re accessing the data programmatically, that won’t work nicely
-          for you because the data is hosted on github, which doesn’t serve the
-          right MIME type when your application fetches the data. Instead, you
-          should pull it via RawGit, and to do that you’ll need to know the URL of
-          the most recent data &mdash; and that means determining the SHA of the git
-          commit that put it there. The SHA may be different for each legislature
-          (because we rebuild the data as changes and additions come in on a
-          per-legislature basis), so to find that out, you need to look in
-          <code>countries.json</code>.
+          But if you’re accessing the data programmatically, that might not work
+          nicely for you because the data is hosted on GitHub, which doesn’t
+          serve the right Content-type when your application fetches the data.
+          Instead, <a href="use_the_data.html#rawgit">you should get it from
+          <code>rawgit.com</code></a> (while you’re there, you should also read
+          our notes on <a href="use_the_data.html">using the data</a> in
+          general). To do that you’ll need to know the URL of the most recent
+          data &mdash; and that means determining the SHA of the git commit
+          that put it there. You get that SHA from the current
+          <code>countries.json</code>, as shown below.
         </p>
+        
+        <h3>
+          Example URLs
+        </h3>
+        <p>
+          This is what an EveryPolitician data URL looks like:
+        </p>
+        <p class="url-demo-box">
+          <code><span class="url-part-domain">http://<em>domain</em></span><br>
+          &nbsp;&nbsp;
+          /everypolitician/everypolitician-data<br>
+          &nbsp;&nbsp;
+          /<span class="url-part-sha">commit-SHA</span><br>
+          &nbsp;&nbsp;
+          /<span class="url-part-file">path-to-data-file</span></code>
+        </p>
+        <p>
+          You might expect the domain to be <code>http://github.com/</code>, but for
+          your application <a href="/use_the_data.html#rawgit">we recommend you use <code><span class="url-part-domain">http://rawgit.com</span></code></a>
+          (or for best performance: <code><span class="url-part-domain">http://cdn.rawgit.com</span></code>).
+        </p>
```

Comment:
I don't think we should ever direct people to "rawgit.com" directly (rather than cdn.rawgit.com). As per the "When people misuse rawgit.com, it costs me money. Please don't be a jerk" part of the [FAQ](http://rawgit.com/faq) you link to in the other section, using the cdn.rawgit version isn't just a "for best performance" nicety — so it should probably be the _only_ version we suggest.